x-pack/legacy/plugins/canvas/public/components/editor/README.md

```diff
@@ -0,0 +1,13 @@
+# Editor Component
+
+This re-usable code editor component was built as a layer of abstraction on top of the Monaco Code Editor (and the React Monaco Editor component). The goal of this component is to expose a set of the most-used, most-helpful features from Monaco in a way that's easy to use out of the box. If a use case requires additional features, this component still allows access to all other Monaco features.
+
+This editor component allows easy access to:
+* Syntax highlighting (including custom language highlighting)
+* Suggestion/autocompletion widget
+* Function signature widget 
+* Hover widget
+
+_TODO: Examples of each_
```

Comment:
nit: I'd create an issue to add these examples, and link it here, so we don't lose track of it.  I don't think this is a blocker to merging.